src/ucs/sys/sys.c

```diff
@@ -1354,6 +1354,16 @@ int ucs_sys_getaffinity(ucs_sys_cpuset_t *cpuset)
     return ret;
 }
 
+ucs_status_t ucs_sys_pthread_getaffinity(ucs_sys_cpuset_t *cpuset)
+{
+    if (pthread_getaffinity_np(pthread_self(), sizeof(*cpuset), cpuset)) {
+        ucs_error("sched_getaffinity() failed: %m");
```

Comment:
1. It's `pthread_getaffinity_np`.
2. Do we need this `ucs_error` here at all? It also exists in the calling place.